ecs-cli/modules/compose/ecs/utils/convert_task_definition.go

```diff
@@ -143,6 +143,9 @@ func convertToContainerDef(inputCfg *libcompose.ServiceConfig,
 	outputContDef.DockerSecurityOptions = aws.StringSlice(inputCfg.SecurityOpt)
 	outputContDef.EntryPoint = aws.StringSlice(inputCfg.Entrypoint.Slice())
 	outputContDef.Environment = environment
+	if inputCfg.Restart == "no" {
```

Comment:
Thanks a lot for the pull request.

Compose options that ECS supports are whitelisted in [this file](https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/compose/ecs/utils/convert_compose_yml.go#L29). You will have to explicitly add "restart" there.
Can you also add a unit test for this change?